iina/Utility.swift

```diff
@@ -461,7 +461,7 @@ class Utility {
     guard let contents = try? FileManager.default.contentsOfDirectory(
       at: folder,
       includingPropertiesForKeys: [.creationDateKey],
-      options: .skipsSubdirectoryDescendants) else { return nil }
+      options: .skipsSubdirectoryDescendants), !contents.isEmpty else { return nil }
```

Comment:
```suggestion
      options: .skipsSubdirectoryDescendants),
      let latestFile = contents.first else { return nil }
```

Reply:
Fixed. I did have to use var instead of let as the code below this updates latestFile.

Reply:
Ah, I see. I would probably replace the code below to use `min(by:)` with a custom comparator, to be honest, but it's up to you if you want to change that here.

Reply:
That was existing code that I thought could be simplified, but I chose to make the minimum change to fix the issue. I have just pushed a commit that simplifies this method. See if that is what you were expecting.